ReactNative/ReactTwitter/android/app/src/main/java/com/reacttwitter/MainActivity.java

```diff
@@ -40,20 +40,21 @@ protected boolean getUseDeveloperSupport() {
      */
     @Override
     protected List<ReactPackage> getPackages() {
+        pushNotificationPackage = new ReactNativePushNotificationPackage(this);
         return Arrays.asList(
                 new MainReactPackage(),
-                oauthIntentPackage
+                oauthIntentPackage,
+                pushNotificationPackage
         );
     }
 
     @Override
     protected void onNewIntent(Intent intent) {
         super.onNewIntent(intent);
         Uri data = intent.getData();
-        if (data == null || !REACT_TWITTER_OAUTH_SCHEME.equals(data.getScheme())) {
-            return;
-        }
 
         oauthIntentPackage.handleOnNewIntent(data);
+
+        pushNotificationPackage.newIntent(intent);
```

Comment:
It passes the `Intent` to the `ReactNativePushNotificationPackage`, to deep link the notification. Unfortunately, the method's name is not the best, but I am not in control of that